lib/broadway_cloud_pub_sub/options.ex

```diff
@@ -13,7 +13,7 @@ defmodule BroadwayCloudPubSub.Options do
     # Handled by Broadway.
     broadway: [type: :any, doc: false],
     client: [
-      type: :atom,
+      type: {:or, [:atom, :mod_arg]},
       default: BroadwayCloudPubSub.PullClient,
       doc: """
       A module that implements the BroadwayCloudPubSub.Client behaviour.
```

Comment:
Perhaps we should document the options of the default PullClient here? And also how to enable/disable/configure retries. :)